### PR TITLE
Fixed for newer DiffEqBase

### PR DESCRIPTION
**Pull request by Stamp1993**
After DiffEqBase update DASSL stopped returning dus, so it worked only with versions 5.15.0, I fixed it in this update

### CODE REVIEW

File: src/common.jl [hunk 1]
```diff
@@ -42,6 +42,8 @@ function solve(
                                 dy0 = prob.du0,
                                 maxorder=alg.maxorder,
                                 factorize_jacobian=alg.factorize_jacobian)
+
+    println("dus = $dus")
```
Comment (ChrisRackauckas) [inline src/common.jl]:
```suggestion
```

File: src/common.jl [hunk 2]
```diff
@@ -54,6 +56,6 @@ function solve(
         end
     end
     =#
-    DiffEqBase.build_solution(prob,alg,ts,timeseries,du=dus,
```
Comment (ChrisRackauckas) [inline src/common.jl]: This doesn't seem to be correct. It needs `du= ...`

https://github.com/JuliaDiffEq/DiffEqBase.jl/blob/master/src/solutions/dae_solutions.jl#L18

Comment (Stamp1993) [reply src/common.jl]: when there was du= build_solution replaced du by nothing, don't sure why, but now it doesn't. That's all the fix, actually

File: src/common.jl (hunk 1 above)
Comment (Stamp1993) [reply src/common.jl]: Sorry, forgot to remove it

**Comment by Stamp1993** [comment]
There was one more problem with callbacks

**Pull request closed without merging**